WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/bloggingprompts/BloggingPromptCardViewModelSlice.kt

```diff
@@ -115,6 +194,44 @@ class BloggingPromptCardViewModelSlice @Inject constructor(
         }
     }
 
+
+    /**
+     * This function is used to make sure the [refresh] information is propagated and processed correctly even though
+     * the previous status is still the current one. This avoids issues like the loading progress indicator being shown
+     * indefinitely.
+     *
+     * Also, for this card source, this can be used as the error state as we don't have any special error handling at
+     * this point, so we just show the last available prompt.
+     */
+    private fun postLastState() {
+        _isRefreshing.postValue(false)
+        //do nothing WIP @ajesh: remove this once this case doesnt occur on the new architecture
```

Comment:
@AjeshRPai - Not entirely sure if this is a leftover comment or work still needs to be done.